tests/cli/Cli/Task/GetEventsManagerCest.php

```diff
@@ -25,7 +27,9 @@ class GetEventsManagerCest
     public function cliTaskGetEventsManager(CliTester $I)
     {
         $I->wantToTest('Cli\Task - getEventsManager()');
-
-        $I->skipTest('Need implementation');
+        $eventsManager = new Manager();
+        $task = new Task();
+        $task->setEventsManager($eventsManager);
```

Comment:
```
[Error] Call to undefined method Phalcon\Cli\Task::setEventsManager()
```
See: https://github.com/phalcon/cphalcon/pull/14445

Reply:
Will rebuild first. Looks like I'm missing this locally

Reply:
Starting to wonder if we even should have this methods here. Maybe we can just remove the test.

Reply:
It makes sense to keep it because we need to have tests for CLI events.